code/datums/diseases/advance/symptoms/fire.dm

```diff
@@ -45,13 +45,13 @@ Bonus
 	return
 
 /datum/symptom/fire/proc/Firestacks_stage_4(mob/living/M, datum/disease/advance/A)
-	var/get_stacks = (sqrtor0(20+A.totalStageSpeed()*2))-(sqrtor0(16+A.totalStealth()))
+	var/get_stacks = max((sqrtor0(20 + A.totalStageSpeed() * 2)) - (sqrtor0(16 + A.totalStealth())), 1)
 	M.adjust_fire_stacks(get_stacks)
-	M.adjustFireLoss(get_stacks/2)
+	M.adjustFireLoss(get_stacks / 2)
```

Comment:
`* 0.5`

Reply:
can also do >> 1. But *.5 is gut enough

Reply:
the point is purely code related.

Multiplication is faster than division, so, where possible, utilize it (not always possible, I realize).

Reply:
Aye multiplication is faster. bitwise should be faster as well but I'm not sure if it'll work in DM due to it using floats/doubles